test/README

```diff
@@ -59,6 +61,8 @@ First, you need to run ./testsuite-prepare.  This will download
 necessary images, compile cockpit, install the result, and set up some
 virtual networking.  It might ask for the sudo password in order to
 setup the network.
```

Comment:
I added a line for the paragraph about mock. The statement `First, you need to run ./testsuite-prepare.` is more of an introduction and shouldn't be separated from the explanation of the process, I think.